src/Tokenizers/Tokenizer.php

```diff
@@ -27,7 +27,7 @@ abstract class Tokenizer
      *
      * @var string
      */
-    protected $eolChar = [];
+    protected $eolChar = '';
```

Comment:
Should the default value be non-empty? What are your thoughts on using `PHP_EOL` here instead of an empty string. Or perhaps `null` or unset.

Reply:
Thanks for the explanation. This all makes sense. :+1: